doc/articles/contributing/guidelines/breaking-changes.md

```diff
@@ -47,7 +47,7 @@ You can find the report within the `NuGetPackages.zip` archive on the build serv
 
 Key things to pay attention to:
 - the report is for each TFM and the assemblies for that TFM.
-- breaking changes which have been blessed have been ~~striked out~~.
+- breaking changes which have been blessed have been ~~struck out~~.
 - `Windows.UI.Xaml.ResourceDictionary::MergedDictionaries()` is the breaking change which has not been blessed and thus the process failed the build.
 
 ## MonoAndroid80 Platform
```

Comment:
from this line till the end, it is an example of output (yes, it is in markdown)
which should be wrapped between:
~~~
```md
```
~~~
and, anything in between should be kept intact

Reply:
In this case I believe I only modified the spelling of the word, so as to keep to the purpose of the PR. If you'd like me to make an additional change, I'm happy to consider it.

Reply:
on lines 301 and 370, you added `inline-code-snippet`; these are part of said output
if you could, can you please wrap L53~EOF in code snippet and undo the changes in between those lines? thanks

Reply:
✅  @Xiaoy312 thanks for clarifying! I made the requested changes.